src/Sylius/Behat/Service/NotificationValidatorInterface.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+namespace Sylius\Behat\Service;
+
+/**
+ * @author Łukasz Chruściel <lukasz.chrusciel@lakion.com>
+ */
+interface NotificationValidatorInterface
+{
+    /**
+     * @param string $resource
```

Comment:
So that we can define that it really gives a feedback, right now it shouldn't throw any exception or return anything. `@throws NotificationNotFoundException`.